2.4/contrib/etc/mongodb.conf

```diff
@@ -0,0 +1,21 @@
+# mongodb.conf
+
+port = 27017
+#fork = true
+pidfilepath = /opt/rh/mongodb24/root/var/run/mongodb/mongodb.pid
+dbpath = /var/lib/mongodb
+
+# Disable data file preallocation.
+noprealloc = true
```

Comment:
yeah the cough is in the right place here :P
Will think about possible solutions in here and maybe it would be a good topic for arch meeting. It would be certainly beneficial if users could supply the config files for the DBs.